src/org/zaproxy/zap/extension/alertFilters/ContextAlertFilterPanel.java

```diff
@@ -103,7 +103,8 @@ public AlertFiltersMultipleOptionsPanel(ExtensionAlertFilters extension, AlertFi
 			super(model);
 			this.extension = extension;
 
-			getTable().getColumnExt(0).setMaxWidth(70);
+			getTable().getColumnExt(0).sizeWidthToFit();
```

Comment:
This statement might have no effect. It shouldn't by default, since there's no custom renderer being set. Also, when it has effect, it doesn't allow to reduce its size :/

How about to just set the max width, using the max width of the "renderer" (if it has one otherwise use the default, `getTable().getTableHeader().getDefaultRenderer()`)?